src/pages/login.vue

```diff
@@ -1,23 +1,20 @@
 <template>
-    <div class="background uk-background-primary">
-
-        <div>
-            <img src="core/gfx/cloud-logo-invert.svg" width="50%">
-        </div>
-        <div>
-            <div class="uk-position-center uk-background-muted uk-border-rounded uk-padding">
-                <h2 v-translate>
-                    Welcome to ownCloud
-                </h2>
-                <div v-translate class="uk-text-small">
-                    Please click the button below to authenticate with ownCloud and get access to your data.
-                </div>
-                <div class="uk-padding">
-                    <button id="authenticate" class="uk-button uk-button-primary" @click="authenticate" v-translate>Authenticate</button>
-                </div>
-            </div>
-        </div>
-    </div>
+    <v-layout class="background primary">
+        <v-img
+          :src='`${this.$store.state.config.theme.logo.big}`'
+          width="50%"
+          aspect-ratio="1.96">
+          <v-flex class="theme--light grey lighten-2 pa-3 ma-3" xs4>
```

Comment:
these classes are not that easy to understand compared to uikit ...

Reply:
that's true but im sure this isn't the cleanest solution to make this working im just bad at this.